printing/ppconstr.mli

```diff
@@ -76,6 +76,15 @@ val default_term_pr : term_pr
 
 val lsimpleconstr : entry_relative_level
 val ltop : entry_relative_level
+
+(* Print at level "simpleconstr"  (applications are surrounded with parentheses)
+   ensured not to be overriden, on the contrary of pr_constr_expr *)
```

Comment:
Does anything actually use the overriding mechanism? I don't see anything in https://github.com/search?q=set_term_pr+language%3AOCaml+&type=code&p=1 
If it's not used we could remove it instead of having to document which APIs can get overridden.

Reply:
Indeed I thought the mechanism was already removed.

I think the intent was to help UIs, but we have taken a different path these days.